layers/generate_layers.py

```diff
@@ -382,7 +384,8 @@ def makeLocalAuthorityDistricts():
 
         # Only keep England
         gj["features"] = list(
-            filter(lambda f: f["properties"]["LAD23CD"][0] == "E", gj["features"])
+            filter(lambda f: f["properties"]["LAD23CD"]
```

Comment:
Out of curiosity, did you use a formatter or your IDE on the file? I've been running black (installed from another project that uses Poetry for dependency management). If this code keeps growing, we can set that up